app/templates/events/view/sessions.hbs

```diff
@@ -0,0 +1,103 @@
+<div class="ui basic buttons">
+  <button class="ui button">All</button>
+  <button class="ui button">Pending</button>
+  <button class="ui button">Accepted</button>
+  <button class="ui button">Confirmed</button>
+  <button class="ui button">Rejected</button>
+</div>
+<button class="ui teal button">Create Session</button>
+<select class="ui dropdown">
```

Comment:
Only text. Because technically as far as UX is concerned, export is a single option. Export is the function. The csv and pdf selection is the second step. So it shouldn't be shown as dropdown using an dropdown icon. 

So instead of dropdown icon, add a `download` icon. Be sure to use `right labelled icon` class in the button